db/migrations/20210111171826_create_licences.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const tableName = 'licences'
+
+exports.up = async function (knex) {
+  await knex
+    .schema
+    .createTable(tableName, table => {
+      // Primary Key
+      table.uuid('id').primary().defaultTo(knex.raw('gen_random_uuid()'))
+
+      // Data
+      table.uuid('invoice_id').notNullable()
+      table.string('line_attr1').notNullable() // Licence number is represented as line_attr1
```

Comment:
Using the name `line_attr_1` is a requirement in `transactions`. I think as the concept of a `licence` record is new to the CMA and not something we have to worry about in the TCM we can use the proper, or at least something more meaningful in this table.

So, for example, we could use `licence_number` or `reference` instead of `line_attr_1`. What do you think?